packages/cursorless-everywhere-talon-core/src/types/std.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module "std" {
```

Comment:
why do you need this?

Reply:
We don't anymore. We did in the implementation that read the Cursorless mode environment variable. I'm removing it.